sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaPruning.scala

```diff
@@ -33,6 +32,8 @@ import org.apache.spark.sql.types.{ArrayType, DataType, MapType, StructField, St
  * SQL column, and a nested Parquet column corresponds to a [[StructField]].
  */
 private[sql] object ParquetSchemaPruning extends Rule[LogicalPlan] {
```

Comment:
Ditto for execution module as well as of [SPARK-16964](https://issues.apache.org/jira/browse/SPARK-16964).